docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-1a.md

```diff
@@ -8,22 +8,22 @@ hosting cluster. If instead you are running these controllers as bare
 processes then launch this controller as follows.
 
 ```shell
-# TODO: mailbox controller has kcp dependencies. Will remove when controllers support spaces.
-kubectl ws root:espw
-mailbox-controller -v=2 &
-sleep 20
+ESPW_SPACE_CONFIG="${PWD}/temp-space-config/spaceprovider-default-espw"
+kubectl-kubestellar-get-config-for-space --space-name espw --provider-name default --sm-core-config $SM_CONFIG --output $ESPW_SPACE_CONFIG
```

Comment:
It is more conventional to invoke a kubectl plugin as a kubectl subcommand; that would be `kubectl kubestellar get config for space` in this case.